chain/network/src/peer/peer_actor.rs

```diff
@@ -116,7 +115,11 @@ pub(crate) struct PeerActor {
     /// a given encoding right away.
     force_encoding: Option<Encoding>,
 
-    /// Shared state of the connection. Populated once the connection is established.
+    /// Peer status.
+    peer_status: PeerStatus,
+    /// Peer id and info. Present when ready.
+    peer_info: DisplayOption<PeerInfo>,
```

Comment:
Hm, so this is also used in `other_peer_id`, and I think we might want to adjust that function to continue to return `Some` in outbound+connectiong state, so that it's more obvious that we don't break any existing `other_peer_id.unwrap()`

Reply:
comment fixed